scheduler/utilization_based_host_allocator.go

```diff
@@ -26,18 +26,17 @@ type TaskGroupData struct {
 
 func UtilizationBasedHostAllocator(ctx context.Context, hostAllocatorData HostAllocatorData) (int, error) {
 	distro := hostAllocatorData.Distro
-	if len(hostAllocatorData.ExistingHosts) >= distro.PoolSize {
+	nExistingHosts := len(hostAllocatorData.ExistingHosts)
```

Comment:
nit I'd say "numExistingHosts" rather than "nExistingHosts"

Reply:
There are other new variables in this change with the same convention so probably should extend to all of them ^

Reply:
Will use the num`X` naming convention for all relevant variables.